data_juicer/format/formatter.py

```diff
@@ -262,4 +262,6 @@ def load_formatter(dataset_path,
 
     # no data
     else:
-        raise NotImplementedError
+        raise ValueError('Can not found local data or huggingface '
+                         'dataset-hub for your given path: '
+                         f'{dataset_path} and suffixes: {suffixes}')
```

Comment:
This info conflicts with PR #64. Need to be merged later.

Reply:
Will pay more attention the formatter.py in PR #64